app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function does not validate the input command to ensure it contains exactly three parts and starts with 'cp'. According to the checklist, you should check that the input string contains 3 elements separated by spaces and has 'cp' as the first part.